manifests/g/Geeks3D/MSIKombustor4/4.1.15.0/Geeks3D.MSIKombustor4.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.MSIKombustor4
+PackageVersion: 4.1.15.0
+PackageLocale: en-US
+Publisher: MSI / Geeks3D
+PackageName: MSI Kombustor 4

```

Comment:
```suggestion
PackageName: MSI Kombustor
```

Reply:
> It seems a bit odd-looking to me. I was going back and forth between Geeks3D.MSIKombustor and Geeks3D.MSIKombustor4

Knowing what we do with other installers, I would probably vouch for the former.

If Control Panel showed `MSI Kombustor 4 version 4.1.15.0` then we could probably go for `Geeks3D.MSIKombustor.4` with `MSI Kombustor 4` as PackageName.

At the moment, Control Panel shows `MSI Kombustor 4.1.15.0` so if we remove the version, it'll be `MSI Kombustor`

If you want to add older versions of MSI Kombustor, then you could split it up into `Geeks3D.MSIKombustor.4`; `Geeks3D.MSIKombustor.3`, etc.